src/backend/distributed/commands/dependencies.c

```diff
@@ -85,7 +85,7 @@ EnsureDependenciesExistOnAllNodes(const ObjectAddress *target)
 	 * either get it now, or get it in master_add_node after this transaction finishes and
 	 * the pg_dist_object record becomes visible.
 	 */
-	List *workerNodeList = ActivePrimaryWorkerNodeList(RowShareLock);
+	List *workerNodeList = ActivePrimaryNonCoordNodeList(RowShareLock);
```

Comment:
Usually we try not to use abbreviations. So maybe instead use:
`ActivePrimaryNodeListWithoutCoordinator` or `ActivePrimaryNonCoordinatorNodeList`.

Reply:
Same for `NodeCount` obviously.

Reply:
s/NonCoord/NonCoordinator/g

applied.